plugins/quetz_current_repodata/quetz_current_repodata/main.py

```diff
@@ -0,0 +1,67 @@
+import bz2
+import hashlib
+import json
+from datetime import datetime, timezone
+
+from conda_build.index import _build_current_repodata
+from fastapi import APIRouter
+
+import quetz
+
+router = APIRouter()
```

Comment:
unused?

Reply:
Yeah, I removed it before but then the below `register_router` hook needs it. OR can I remove the hook below as well?

Reply:
I think so :)

Reply:
Done!